### PR TITLE
TW-615 NFT details, Alien, button Buy for x.xx XTZ

### PR DESCRIPTION
**Pull request by svjatoslavtt**

### CODE REVIEW

File: src/hooks/use-buy-collectible.hook.ts [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+  const selectedRpc = useSelectedRpcUrlSelector();
+  const tezos = createTezosToolkit(selectedRpc);
+
+  const selectedAccount = useSelectedAccountSelector();
+  const collectibles = useCollectiblesListSelector();
+  const dispatch = useDispatch();
+  const { navigate } = useNavigation();
+
+  const isUserOwnerCurrentCollectible = useMemo(
+    () => !!collectibles.find(ownCollectible => getTokenSlug(ownCollectible) === getTokenSlug(collectible)),
```
Comment (lendihop) [inline src/hooks/use-buy-collectible.hook.ts:43]: use Boolean instead of !!

File: src/mocks/react-native-firebase-app-check.mock.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+export const mockReactNativeFirebase = {
```
Comment (lendihop) [inline src/mocks/react-native-firebase-app-check.mock.ts:1]: delete this file. We have this mock in react-native-firebase.mock.ts

File: src/modals/collectible-modal/collectible-modal.tsx [hunk 3]
```diff
@@ -64,18 +67,35 @@ export const CollectibleModal = () => {
...
   const iconSize = width - formatSize(32);
 
-  const { navigate } = useNavigation();
+  const collectibles = useCollectiblesListSelector();
 
-  const burnCollectible = useBurnCollectible(collectible);
+  const isUserOwnerCurrentCollectible = useMemo(
+    () => !!collectibles.find(ownCollectible => getTokenSlug(ownCollectible) === getTokenSlug(collectible)),
+    [collectible, collectibles]
+  );
```
Comment (lendihop) [inline src/modals/collectible-modal/collectible-modal.tsx:75]: extract in a separate hook and use where appropriate

File: src/apis/objkt/queries.ts [hunk 4]
```diff
@@ -41,6 +41,16 @@ export const buildGetCollectibleByAddressAndIdQuery = (address: string, tokenId:
           name
         }
       }
+      lowest_ask
+      listings_active(order_by: {price: asc}) {
```
Comment (Boltman92) [inline src/apis/objkt/queries.ts:45]: maybe you need to order by price_xtz to avoid confuses with different currecies

**Review by Boltman92 — APPROVED** [review]

**Review by vladvn1905 — APPROVED** [review]
ios done 1.15_727

**Review by IrynaKhyzhynska — APPROVED** [review]
android done 1.15_727

**Pull request merged**